src/ray/common/task/scheduling_resources.cc

```diff
@@ -190,8 +190,10 @@ const ResourceSet ResourceSet::GetNumCpus() const {
 }
 
 std::string format_resource(std::string resource_name, double quantity) {
-  if (resource_name == "object_store_memory" || resource_name == "memory") {
-    return std::to_string(quantity / (1024 * 1024 * 1024)) + " GiB";
+  if (resource_name == "object_store_memory" ||
+      resource_name.find(kMemory_ResourceLabel) == 0) {
```

Comment:
Nit: maybe boost starts_with?

Btw, could you put some comments here about what's the kind of resource you'd like to convert?